adapter-cqld4/src/main/resources/activities/baselines/graph-wheels.yaml

```diff
@@ -78,7 +78,7 @@ blocks:
            .create()
   dev-mode:
     tags:
-      phase: dev-mode
+      block: dev-mode
```

Comment:
same, already provided, but you can scan for these, I won't comment on the rest.

Reply:
removed